src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java

```diff
@@ -34,6 +36,8 @@ public void testSignUp() {
 
 		//Assert of alert label
 		Assert.assertEquals(getDriver().switchTo().alert().getText(), "Sign up successful.", "Alert is wrong");
+		ScreenshotService screen = new ScreenshotService();
```

Comment:
you can make it in 1 line: `new ScreenshotService().makeScreenshot();`

Also if you want to declare it here do not shorten it to "screen", name it fully "screenshotService"
when you're working in the team we should write code that is clear for others